pkg/auth/transport.go

```diff
@@ -31,6 +31,12 @@ func (t *transport) RoundTrip(req *http.Request) (*http.Response, error) {
 	}
 
 	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", jwt))
+	fmt.Sprintf(
+		"%s,%s",
+		fmt.Sprintf("%s=%s", "sdk_version", "0.0.6"),
+		fmt.Sprintf("%s=%s", "sdk_language", "go"),
+	)
+	req.Header.Set("Correlation-Context", fmt.Sprintf("Bearer %s", jwt))
```

Comment:
Should this be:
```suggestion

	req.Header.Set(
		"Correlation-Context", 
		fmt.Sprintf(
			"%s,%s",
			fmt.Sprintf("%s=%s", "sdk_version", "0.0.6"),
			fmt.Sprintf("%s=%s", "sdk_language", "go"),
		),
	)
```

Reply:
Do we have a way of dynamically seeting the `sdk_version`?

Reply:
Not really, I looked into ways to pull it, but its a lot of go "debug" packages, or oddities. I think once we have a solid `make version` type of script we can just append it there, but until then we can do this scrappy way with the PR comment!